hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/handle/record/RecordStreamBuilder.java

```diff
@@ -1123,6 +1123,18 @@ public ContractOperationStreamBuilder testForIdenticalKeys(@NonNull final Predic
         return this;
     }
 
+    @Override
+    public EvmTransactionResult getEvmTransactionResult() {
+        requireNonNull(contractFunctionResult);
+        return new EvmTransactionResult(
+                contractFunctionResult.senderId(),
+                contractFunctionResult.contractID(),
+                contractFunctionResult.contractCallResult(),
+                contractFunctionResult.errorMessage(),
+                contractFunctionResult.gasUsed(),
+                null); //  Do we need to construct this?
```

Comment:
It seems we _actually_ need just the `ContractFunctionResult#contractCallResult` field here, what if we change the `StreamBuilder` parent method to just, `Bytes getEvmCallResult()` and return `contractFunctionResult.contractCallResult()` here in `RecordStreamBuilder`, `EvmTransactionResult#resultData()` in `BlockStreamBuilder`?